GW2EIBuilders/HtmlModels/HtmlMetaData/SkillDto.cs

```diff
@@ -1,61 +1,93 @@
-﻿using System.Collections.Generic;
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
 using GW2EIEvtcParser;
 using GW2EIEvtcParser.EIData;
 using GW2EIEvtcParser.ParsedData;
 
-namespace GW2EIBuilders.HtmlModels.HTMLMetaData
+namespace GW2EIBuilders.HtmlModels.HTMLMetaData;
+
+
+[JsonConverter(typeof(SkillDataConverter))]
+public struct SkillData2
```

Comment:
That's not the best naming =D Maybe SerializableSkillDto?

Reply:
I'm all for good type names, and have a fair share of gripes towards the codebase around that myself, but that is not it chief. 

How about `SkillCastDto`  ?

Reply:
Yeah, go for it.